spec/unit/cli_spec.rb

```diff
@@ -178,6 +178,17 @@ def check_command(command, parents = [])
 
   include_context 'cli'
 
+  ## TODO: when rspec-mocks fixes #1306, delete this block
```

Comment:
We should be able to drop this once #1047 is merged.

Reply:
I think this error is unrelated to what #1047 fixes.